### PR TITLE
Add implementation for point fields

### PR DESCRIPTION
**Pull request by shawnfeldman**
Adding implementation for point fields

### CODE REVIEW

File: luwak/src/main/java/uk/co/flax/luwak/queryparsers/StandardLuceneQueryParser.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+import org.apache.lucene.analysis.standard.StandardAnalyzer;
+import org.apache.lucene.queryparser.flexible.standard.StandardQueryParser;
+import org.apache.lucene.queryparser.flexible.standard.config.PointsConfig;
+import org.apache.lucene.search.Query;
+import uk.co.flax.luwak.MonitorQueryParser;
+
+/**
+ * A query parser that uses the default Lucene parser
+ */
+public class StandardLuceneQueryParser implements MonitorQueryParser {
```
Comment (romseygeek) [inline luwak/src/main/java/uk/co/flax/luwak/queryparsers/StandardLuceneQueryParser.java]: Could we change this to 'FlexibleLuceneQueryParser', as it's using the flex parser under the hood?

File: luwak/src/test/java/uk/co/flax/luwak/TestMonitorStandardQueryParser.java [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+        .hasMatchCount("doc1", 1);
+
+        mq  = new MonitorQuery("query1","age:1");
+        monitor.update(mq);
+        matches = monitor.match(doc, SimpleMatcher.FACTORY);
+        assertThat(matches)
+        .hasMatchCount("doc1", 1);
+    }
+
+    @Test
```
Comment (romseygeek) [inline luwak/src/test/java/uk/co/flax/luwak/TestMonitorStandardQueryParser.java]: I think we can remove everything below here?  It's just duplicating the tests in TestMonitor, right?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,2 @@
+FROM maven:3.5-jdk-8
```
Comment (romseygeek) [inline Dockerfile]: I'd like to avoid docker dependencies if possible, can you strip these bits out?

**Comment by shawnfeldman** [comment]
pushed again

**Comment by shawnfeldman** [comment]
i rebased my changes so it'll be in a single commit

**Comment by romseygeek** [comment]
Thanks!

**Pull request merged**